apm-sniffer/apm-sdk-plugin/spring-plugins/mvc-annotation-commons/src/main/java/org/apache/skywalking/apm/plugin/spring/mvc/commons/interceptor/AbstractMethodInterceptor.java

```diff
@@ -208,4 +218,32 @@ private void collectHttpParam(HttpServletRequest request, AbstractSpan span) {
             Tags.HTTP.PARAMS.set(span, tagValue);
         }
     }
+
+    private boolean shouldCollectHeader() {
+        List<String> includeHeaders = SpringMVCPluginConfig.Plugin.Http.INCLUDE_HTTP_HEADERS;
+        return !CollectionUtil.isEmpty(includeHeaders) ;
+    }
+
+    private void collectHttpHeaders(HttpServletRequest request, AbstractSpan span) {
+        final Enumeration<String> headerNames =  request.getHeaderNames();
+        if (headerNames == null) {
```

Comment:
I mean check `SpringMVCPluginConfig.Plugin.Http.INCLUDE_HTTP_HEADERS`, because, you were trying to collect header in profiling, which has been removed. 

I think `headerNames` can't be null, right?

Reply:
I have removed the profiling collect header logic, because we can't collect all headers when INCLUDE_HTTP_HEADERS is null

Reply:
Yes, I know. So I said `you were trying to collect header in profiling, which has been removed.`.

This `== null` checker is pointless. This is the reason I posted a comment here.

Reply:
got it. will change

Reply:
done